xbmc/pvr/timers/PVRTimerInfoTag.cpp

```diff
@@ -43,7 +43,7 @@ CPVRTimerInfoTag::CPVRTimerInfoTag(bool bRadio /* = false */) :
   m_bFullTextEpgSearch(false)
 {
   m_iClientId           = g_PVRClients->GetFirstConnectedClientID();
-  m_iClientIndex        = -1;
+  m_iClientIndex        = static_cast<unsigned int>(-1);
```

Comment:
@fritsch , i didn't read all code about but i think this value isn't really important and it is evaluated only in default ctor.